docs/conf.py

```diff
@@ -71,6 +71,10 @@
 release = dask_image._version.__version__
 # The short X.Y.Z version.
 version = '.'.join(release.split('.')[:3])
+if "dev" in release:
+    display_version = release
```

Comment:
```suggestion
    display_version = "(under development)"
```

Reply:
I've changed your suggestion from `"(under development)"` -> `"(development version)"`.

It might just be me, but "under development" makes me think of those old "under construction" website pages that suggest the website is unfinished.

Reply:
Sounds reasonable 🙂